### PR TITLE
EVG-3279 link to display task from execution task

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: service/task.go [hunk 1]
```diff
@@ -297,6 +298,9 @@ func (uis *UIServer) taskPage(w http.ResponseWriter, r *http.Request) {
 		for _, tr := range projCtx.Context.Task.LocalTestResults {
 			uiTask.TestResults = append(uiTask.TestResults, uiTestResult{TestResult: tr})
 		}
+		if uiTask.PartOfDisplay {
+			uiTask.DisplayTaskID = projCtx.Task.DisplayTask.Id
```
Comment (bsamek) [inline service/task.go]: nil checks?

File: service/templates/task.html [hunk 2]
```diff
@@ -239,6 +239,10 @@ <h1 class="one-liner" id="task-title">
                   <td class="icon"><i class="fa fa-ban"></i></td>
                   <td>Blacklisted</td>
                 </tr>
+                <tr ng-show="task.display_task">
+                  <td class="icon"><i class="fa fa-link"></i></td>
+                  <td><a href="/task/[[task.display_task]]">Link to parent display task</a></td>
```
Comment (bsamek) [inline service/templates/task.html]: I don't think links should have the word "link" in them. Maybe just "Parent" or "Parent display task"

File: service/task.go (hunk 1 above)
Comment (john-m-liu) [reply service/task.go]: PartOfDisplay being true guarantees that this is not nil, but I can add them if you'd like

File: service/templates/task.html (hunk 2 above)
Comment (john-m-liu) [reply service/templates/task.html]: changed

**Pull request merged**